docs/rules/space-before-blocks.md

```diff
@@ -116,75 +116,79 @@ class Foo{
 }
 ```
 
-The following patterns are considered problems when configured `{ "functions": "never", "keywords": "always", "classes": "never" }`:
+### functions
```

Comment:
Note: I re-ordered examples so they were in the same order as as the options object:

1. Functions
2. Keywords
3. Classes

Old version was:

1. Keywords
2. Functions
3. Classes